desks-web-app/src/components/booking-components/AvailabilityCard.vue

```diff
@@ -9,7 +9,7 @@
       <v-btn icon="mdi-plus" elevation="0" @click="$emit('addBooking')"></v-btn>
     </template>
     <template v-slot:title>
-      {{ startOfDay.format("dddd") }}
+      {{ startOfDay.isSame(today, "days") ? "Today" : startOfDay.format("dddd") }}
```

Comment:
I am not sure if it is the right way of marking the the current day. For me it makes sense to have a consistent cart title and have a different way of hightailing for the current day. Maybe you can add this information to the sub title and/or change the appearance of the card.